components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -372,40 +424,55 @@ export default class MarketplaceItem extends React.Component {
             );
         }
 
-        const pluginDetailsInner = (
-            <>
-                {this.props.name} <span className='light subtitle'>{versionLabel}</span>
-                {localTag}
-                <p className={classNames('more-modal__description', {error_text: this.props.error})}>
-                    {this.props.error ? this.props.error : this.props.description}
-                </p>
-            </>
-        );
+        let labelsTags;
+        if (this.props.labels && this.props.labels.length !== 0) {
+            labelsTags = this.props.labels.map((label) => (
+                <Label
+                    key={label.name}
+                    name={label.name}
+                    description={label.description}
+                    url={label.url}
+                />
+            )
+            );
+        }
 
-        let pluginDetails;
+        let pluginName;
         if (this.props.homepageUrl) {
-            pluginDetails = (
+            pluginName = (
```

Comment:
I think it's still desirable to have the whole row be clickable. 0/5 on if we need to prevent the labels from being clickable, but it was definitely intentional to provide a "large" clickable surface area to learn more about the plugin, vs having to discover clicking on the name only.

Reply:
Fixed, see https://github.com/mattermost/mattermost-webapp/pull/4547#issuecomment-571109054